### PR TITLE
chromium{,Beta,Dev}: 48.0.2564.97 -> 48.0.2564.116 (release-15.09)

### PR DESCRIPTION
**Pull request by grahamc**
re: https://github.com/NixOS/nixpkgs/pull/13455

From the debian security mailing list:

Several vulnerabilities have been discovered in the chromium web browser.

CVE-2016-1622

```
It was discovered that a maliciously crafted extension could bypass
the Same Origin Policy.
```

CVE-2016-1623

```
Mariusz Mlynski discovered a way to bypass the Same Origin Policy.
```

CVE-2016-1624

```
lukezli discovered a buffer overflow issue in the Brotli library.
```

CVE-2016-1625

```
Jann Horn discovered a way to cause the Chrome Instant feature to
navigate to unintended destinations.
```

CVE-2016-1626

```
An out-of-bounds read issue was discovered in the openjpeg library.
```

CVE-2016-1627

```
It was discovered that the Developer Tools did not validate URLs.
```

CVE-2016-1628

```
An out-of-bounds read issue was discovered in the pdfium library.
```

CVE-2016-1629

```
A way to bypass the Same Origin Policy was discovered in Blink/WebKit,
along with a way to escape the chromium sandbox.
```

(cherry picked from commit 712d59225e52c7636e315cb2cce35e357114b183)
###### Things done:
- [ ] Tested via `nix.useChroot`.
- [ ] Built on platform(s): .
- [ ] Tested compilation of all pkgs that depend on this change.
- [ ] Tested execution of binary products.
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by bendlas** [comment]
cc @aszlig

File: pkgs/applications/networking/browsers/chromium/source/sources.nix [hunk 1]
```diff
@@ -1,21 +1,17 @@
 # This file is autogenerated from update.sh in the parent directory.
 {
   dev = {
-    version = "49.0.2593.0";
-    sha256 = "1krkf8dg10xnvxs4zfvy2rkp0v3gxg55418b11kmfywsxjfi09hv";
-    sha256bin32 = "0sarfj8r6x28pbmcpd5facm6liifazj744c9qn6ci6xg7f8jlnxi";
-    sha256bin64 = "1l5s504mmr4v5y67fsla9hyrk7dm155yc790ld87s1ya03h455gr";
+    version = "50.0.2657.0";
+    sha256 = "1xc66zsjd5snydf60di2k2vzwwcqs1p16yn4jbpj2khm9cmxmkmf";
```
Comment (fpletz) [inline pkgs/applications/networking/browsers/chromium/source/sources.nix]: The `sha256bin64` is missing here and for beta. @aszlig added two to add those here: 716b79d3a5f3ee438942f59bfd879f46e66204ac & 54b49125664adb1f7cf054bbf8a09aeecc88b574. Maybe backport those as well?

**Comment by grahamc** [comment]
Updated to include the bin64's.

**Comment by fpletz** [comment]
Thanks!

**Pull request merged**